src/Tools/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CSharp.Analyzer.Testing/CSharpDiagnosticVerifier.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.CodeAnalysis.Analyzer.Testing;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp.Analyzer.Testing
+{
+    public class CSharpDiagnosticVerifier<TAnalyzer, TVerifier> : DiagnosticVerifier<TAnalyzer, CSharpAnalyzerTest<TAnalyzer, TVerifier>, TVerifier>
+        where TAnalyzer : DiagnosticAnalyzer, new()
+        where TVerifier : IVerifier, new()
+    {
+    }
+
+    public static class CSharpDiagnosticVerifier<TAnalyzer>
```

Comment:
:bulb: One file per top-level type to ensure, among other things, <kbd>t</kbd> operates as Navigate To Type on GitHub.